packages/libs/api/icy-graphql-client/src/queries/nft/getNFTsByWalletAddress/getNFTsByWalletAddress.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from '@apollo/client/core';
+import { PaginationArgs } from '../../../types';
```

Comment:
Would prefer an absolute path here `... from '~/types'` or whatever the path is

Reply:
I had tried using basePath alias imports like https://github.com/quiknode-labs/qn-oss/pull/6/commits/830fefe49cde71240fbf67c6313984b4cec86404#diff-84aee5d7a1e271d85603f9d48b0100bd95edc3757d17a330eb5475dd8c99abe2L6 but it was causing issues when the code was directly reimported on other nx projects like on https://github.com/quiknode-labs/qn-oss/pull/6/files#diff-9d88951a6aa436ee6e66eb18c9215ab66c0f29e9cb8d6d5d674b668c2ddbc61aR1, it might be possible to fix with specific aliases or some top level ts config, but I was planning on punting this as an improvement for now, WDYT?

Reply:
Punting is fine, but would like to be able to use absolute paths at some point here